src/geometries/WireframeGeometry.js

```diff
@@ -175,5 +179,26 @@ function WireframeGeometry( geometry ) {
 WireframeGeometry.prototype = Object.create( BufferGeometry.prototype );
 WireframeGeometry.prototype.constructor = WireframeGeometry;
 
+WireframeGeometry.prototype.toJSON = function () {
+
+	var data = BufferGeometry.prototype.toJSON.call( this );
+	var geometry = data.geometry;
+
+	if ( geometry.isGeometry ) {
+
+		geometry = new BufferGeometry().fromGeometry( geometry );
+
+	} else {
+
+		geometry = geometry.clone();
```

Comment:
This ensures the serialization/deserialization of both geometries is self-contained.